src/main/template/presto.yaml

```diff
@@ -0,0 +1,652 @@
+          services:
+            sysvinit:
+              haveged:
+                enabled: true
+                ensureRunning: true
+        openjdk8:
+          packages:
+            yum:
+              java-1.8.0-openjdk: ["1.8.0.222.b10-0.amzn2.0.1"]
+              java-1.8.0-openjdk-devel: ["1.8.0.222.b10-0.amzn2.0.1"]
+        aws_cli:
+          files:
+            /home/ec2-user/.aws/config:
+              content: !Sub |
+                [default]
+                region=${AWS::Region}
+                output=json
+              owner: ec2-user
+              group: ec2-user
+              mode: "000644"
+        register_dns:
+          commands:
+            01_create_change_batch:
+              command: !Join
+                - ""
+                - - "echo '{\"Changes\": [{\"Action\": \"UPSERT\", \"ResourceRecordSet\": {\"Name\": \"coordinator."
+                  - !Ref internalDomain
+                  - "\", \"Type\": \"A\", \"TTL\": 60, \"ResourceRecords\": [{\"Value\": \"'$(hostname -I)'\"}]}}]}' > /home/ec2-user/change-batch.json"
+            02_register_dns:
+              command: !Join
+                - ""
+                - - "aws route53 change-resource-record-sets --hosted-zone-id "
+                  - !Ref hostedZone
+                  - " --change-batch file:///home/ec2-user/change-batch.json"
+            03_remove_change_batch:
+              command: "rm /home/ec2-user/change-batch.json"
+            04_add_hosts_entry:
+              command: !Join
+                - ""
+                - - "echo \"$(hostname -I) coordinator."
+                  - !Ref internalDomain
+                  - "\" >> /etc/hosts"
+        download_presto:
+          commands:
+            download_presto:
+              command: |
+                #!/bin/bash -xe
+                mkdir /opt/presto
+                wget -q -O - "https://repo1.maven.org/maven2/io/prestosql/presto-server/326/presto-server-326.tar.gz" | tar xvzf - -C /opt/presto
+                mv "/opt/presto/presto-server-326" /opt/presto/presto-server
+                mkdir /var/presto
+                mkdir /var/presto/data
+        configure_presto:
+          files:
+            /opt/presto/presto-server/etc/node.properties:
+              content: |
+                node.data-dir=/var/presto/data
+                node.environment={{nodeEnvironment}}
+              context:
+                nodeEnvironment: !Ref environmentName
+              owner: root
+              group: root
+              mode: "000644"
+            /opt/presto/presto-server/etc/jvm.config:
+              content: |
+                -server
+                -XX:-UseBiasedLocking
+                -XX:+UseG1GC
+                -XX:G1HeapRegionSize=32M
+                -XX:+ExplicitGCInvokesConcurrent
+                -XX:OnOutOfMemoryError="kill -9 %p"
+                -XX:+UseGCOverheadLimit
```

Comment:
According to https://stackoverflow.com/a/45927722/7641, this option is ignored by G1; we should remove it.